src/exercise/05.js

```diff
@@ -4,14 +4,16 @@
 import * as React from 'react'
 import '../box-styles.css'
 
-// 💰 Use the className for the size and style (backgroundColor, fontStyle) for the color and the font style
```

Comment:
this line was especially confusing because the intention is for the `style` prop to be used for these properties, not `className`